games-action/supertuxkart/supertuxkart-1.2.recipe

```diff
@@ -0,0 +1,102 @@
+to create a game that is more fun than realistic, and provide an enjoyable experience \
+for all ages.
+
+Discover the mystery of an underwater world, or drive through the jungles of Val Verde \
+and visit the famous Cocoa Temple. Race underground or in a spaceship, through a rural \
+farmland or a strange alien planet. Or rest under the palm trees on the beach, watching \
+the other karts overtake you. But don't eat the bananas! Watch for bowling balls, \
+plungers, bubble gum, and cakes thrown by your opponents.
+
+You can do a single race against other karts, compete in one of several Grand Prix, \
+or try to beat the high score in time trials on your own. Play battle mode against \
+the computer or your friends, and more! For a greater challenge, race online against \
+players from all over the world and prove your racing skills!"
+HOMEPAGE="https://supertuxkart.net"
+COPYRIGHT="2020 SuperTuxKart Development Team"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/supertuxkart/stk-code/releases/download/$portVersion/supertuxkart-$portVersion-src.tar.xz"
+CHECKSUM_SHA256="052edf0afdbeb99583fe8676fb0ab80ecb6103fb88b7540f858d1b5fa1297d37"
+SOURCE_DIR="SuperTuxKart-$portVersion-src"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	supertuxkart$secondaryArchSuffix = $portVersion
+	app:SuperTuxKart$secondaryArchSuffix
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libenet$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libglu$secondaryArchSuffix
+	lib:libglew$secondaryArchSuffix
+	lib:libharfbuzz$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libopenal$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	lib:libsdl2_2.0$secondaryArchSuffix
+	lib:libsqlite3$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libcurl$secondaryArchSuffix
+	devel:libenet$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libglu$secondaryArchSuffix
+	devel:libglew$secondaryArchSuffix >= 2
+	devel:libharfbuzz$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libpng$secondaryArchSuffix
+	devel:libopenal$secondaryArchSuffix
+	devel:libogg$secondaryArchSuffix
+	devel:libsdl2$secondaryArchSuffix
+	devel:libsqlite3$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libvorbis$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	mkdir -p build
+	cd build
+
+	cmake .. \
+		-DCMAKE_BUILD_TYPE=STKRelease \
+		-DSTK_INSTALL_DATA_DIR=$dataDir/SuperTuxKart \
+		-DSTK_INSTALL_BINARY_DIR=$appsDir/SuperTuxKart \
```

Comment:
This can be done in $appsDir, no need to put it into a subfolder

Reply:
If you can put the info here, I'll add to recipe.... ensuring it works always works in deskbar and menu....

Reply:
´´´diff
- 		-DSTK_INSTALL_BINARY_DIR=$appsDir/SuperTuxKart \ 
+ 		-DSTK_INSTALL_BINARY_DIR=$appsDir \
```